### DISCUSSION TITLE
Termux: Two Years of Silence - Uncertainty Surrounding the Future of the Popular Android Terminal Emulator

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-app#3701] [General]
It has been over two years since Termux, a popular Android terminal emulator and Linux distribution, introduced any significant updates in terms of new features or bug fixes. This prolonged absence of development activity has left many users wondering if the Termux project is losing steam.

The lack of progress in Termux has raised concerns among its loyal user base, as they have grown accustomed to frequent improvements and additions to the software's functionality. Some users are speculating whether this stagnation might be indicative of an imminent demise for the Termux project altogether.

However, while there haven't been any major breakthroughs in recent times, the Termux team has not given up on the platform entirely. They continue to maintain the existing codebase and release minor patches from time to time, addressing critical bugs and security vulnerabilities as they arise.

In light of these developments, it remains unclear exactly where Termux is headed next. While some observers believe that the silence could signify the beginning of the end for the app, others contend that the developers may simply be working behind closed doors on groundbreaking advancements that will eventually revolutionize the platform once again.

Until further notice, we can only wait and see how things unfold for Termux moving forward. Given its popularity and widespread adoption across various industries, it would come as no surprise if the community rallies around the cause and sparks renewed interest in driving innovation within the project. Ultimately, only time will reveal the fate of Termux - until then, let us hope for the best!

**Comment by twaik** [comment]
Development is not stopped. Packages are updated separately from main application...

**Reply by ghost** [reply]
but [termux-app](https://github.com/termux/termux-app) is dead.

**Reply by twaik** [reply]
It is not.
